docs/source/tutorials_source/package/tutorial_custom_augmentations.py

```diff
@@ -222,17 +219,15 @@ def tiff_loader(f):
         return np.array(image)
 
 
-# create the dataset and overwrite the image loader
-dataset_train = LightlyDataset(input_dir=path_to_data)
+# Create the dataset with the custom transform and overwrite the image loader.
+dataset_train = LightlyDataset(input_dir=path_to_data, transform=transform)
```

Comment:
Here we pass the transform now and a few lines below we no longer pass the collate function.